wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/th.md

```diff
@@ -1,76 +1,85 @@
-# Handling Foul Play (ภาษาไทย)
+# การจัดการกับการโกง
 
 ## ทำไมฉันถึงโดนแบนล่ะ
 
-คุณโดนแบนเพราะว่า...
+ผู้ใช้ใด ๆ ก็ตาม อาจโดนแบนถ้าทำบางสิ่งในสิ่งเหล่านี้:
 
-- **โดนใบ้ 10 ครั้ง** หรือ
-- **คุณให้คนอื่นเล่นบัญชีของคุณ** หรือ
-- **สมัครมากกว่าหนึ่งบัญชี** หรือ
-- **โกง** (อ่านข้างล่างต่อ)
+- การทำผิดกฏอย่างต่อเนื่อง ไม่ว่าจะกี่ครั้งก็ตาม
+- หนีการทำโทษโดยการสร้างบัญชีผู้ใช้ใหม่
+- แบ่งปันบัญชีของตัวเองให้ผู้อื่น
+- ใช้หลายบัญชี (multi-accounting)
+- โกง (อ่านข้างล่างต่อ)
+
+## อะไรที่ถือว่าเป็นการใช้หลายบัญชี (multi-accounting)
+
+ผู้ใช้แต่ละคนสามารถใช้ได้แค่**หนึ่ง**บัญชีเท่านั้น การสร้างบัญชีใหม่เป็นสิ่งที่ห้ามในทุกกรณี ความผิดครั้งแรกจะถูกทำโทษด้วยการเงียบ (silence) ที่มีระยะเวลาขึ้นอยู่กับจำนวนบัญชีที่สร้างขึ้น โดยผู้ใช้ที่สร้างบัญชีใหม่ ๆ แม้ว่าถูกเตือนแล้วจะถูกแบน (restricted)
 
 ## อะไรที่ถือว่าเป็นการโกง
 
-ตามที่กฎบอกไว้
+[กฎ](/wiki/Rules)ได้บอกไว้ว่าการแก้ไขตัวไคลเอนต์ (client) ไม่ว่าจะเป็นการโกงคะแนน การทำมาโคร การทำ auto-aim และทุกอย่างที่ไม่ได้เกิดจากการกระทำของผู้เล่นเอง จะถือว่าเป็นการโกง
 
-`การดัดแปลง หรือแก้ไขตัวไคลเอนต์ ไม่ว่าจะเป็นการโกงคะแนน การทำมาโคร การทำ auto-aim และทุกอย่างที่ไม่ได้เกิดจากการกระทำของมนุษย์ (สายพันธฺุ์ `*`Homo` `sapiens`*`) `
-`จะถือว่าเป็นการโกง`
+กฏจะไม่มีการเตือน ผู้ที่โกงจะถูกแบนไม่จำกัดระยะเวลาสำหรับการทำผิดครั้งแรก โดยทางทีมอาจจะตัดสินใจปลดแบนออกถ้าผู้เล่นส่งคำร้องอุทธรณ์ (appeal) สำเร็จ
 
-มีการลงโทษเพียงสถานเดียว ผู้ที่โกงจะถูก **แบนถาวร**เข้าใจตรงกันนะ
+## อะไรกัน โดนแบนทั้ง ๆ ที่ไม่ได้โกงเลย
 
-## อะไรกัน โดนแบนทั้งๆ ที่ไม่ได้โกงเลย
+osu! ใช้ระบบตรวจจับการโกงอัตโนมัติ รวมถึงการใช้มนุษย์ยืนยันว่าโกงจริง หากคุณไม่รู้สาเหตุที่ถูกแบน ให้ติดต่อ [accounts@ppy.sh](mailto:accounts@ppy.sh)
```

Comment:
เพิ่มว่าต้องติดต่อเป็นภาษาอังกฤษด้วยดีไหมฮะ เพราะบางคนเห็นหน้านี้เป็นไทย แล้วอาจส่งภาษาไทยไป

Reply:
อันนี้ไม่แน่ใจอ่ะครับ แต่ที่ไปส่องมาใน zh.md เค้าไม่ได้เขียนอะไรเกี่ยวกับเรื่องภาษานะ เพราะว่าแปลตรงตัวจาก file อังกฤษ

Reply:
อันนี้แล้วแต่เลยฮะ ที่ผมคิดคือใส่ไว้กันคนพิมพ์ไทยไป

Reply:
Reviewer is suggesting that we should add that all emails to accounts@ppy.sh should be in English. I answered that I took a look at zh.md (which is up-to-date) and this clause does not exist on there.

Will leave this open in case someone wants to take a look before merging.